spring-cloud-aws-samples/spring-cloud-aws-sqs-sample/src/main/resources/application.yaml

```diff
@@ -0,0 +1,8 @@
+cloud:
+    aws:
+        region:
+            static: eu-central-1
```

Comment:
if we enforce region here, it should be also enforced in CDK. Or dropped in both places.